src/rendering/SceneManager.cc

```diff
@@ -938,6 +938,7 @@ rendering::LightPtr SceneManager::CreateLight(Entity _id,
   light->SetAttenuationRange(_light.AttenuationRange());
 
   light->SetCastShadows(_light.CastShadows());
+  light->SetIntensity(_light.Intensity());
```

Comment:
just added a comment in ign-msgs to default this to 1.0 so that we don't end up with a value of 0 here if the field is not specified.

Reply:
ok nevermind, this is done in light creation and Light SDF DOM has a default value of 1.0 so this should be fine.